warehouse/manage/views/__init__.py

```diff
@@ -1384,7 +1384,8 @@ def delete_oidc_publisher(self):
             )
 
             self.request.session.flash(
-                f"Removed {publisher} from {self.project.name}", queue="success"
+                f"Removed trusted publisher for project {self.project.name!r}",
```

Comment:
For my own benefit: why are we taking the `repr` of `self.project.name` here? Just to intelligently quote it?